app/questionnaire/questionnaire_store_updater.py

```diff
@@ -200,6 +200,13 @@ def update_section_status(
             section_status, section_id, list_item_id
         )
 
+    def sections_to_evaluate(self, filter_by: Set[str] = None) -> Union[List, None]:
+        completed_sections = self._progress_store.completed_section_keys()
+        try:
+            return list(completed_sections & filter_by)
+        except TypeError:
+            return list(completed_sections)
```

Comment:
I personally prefer `if` when dealing with `NoneType` as that is which is common in this project.
```py
return (
    list(completed_sections)
    if filter_by is None
    else list(completed_sections & filter_by)
)
```

Reply:
Happy to use the ternary in this instance